lib/sierra-models/base.js

```diff
@@ -64,7 +64,7 @@ class SierraBase {
       return { ...lookup, [number]: rawField }
     }, {})
 
-    const lookup = (field) => {
```

Comment:
This is a misleading name for a function that attaches a raw parallel field to the given primary field.

Reply:
👍